eng/common/scripts/Verify-Links.ps1

```diff
@@ -220,14 +220,14 @@ function CheckLink ([System.Uri]$linkUri, $allowRetry=$true)
       $headRequestSucceeded = $true
       try {
         # Attempt HEAD request first
-        $response = Invoke-WebRequest -Uri $linkUri -Method HEAD -UserAgent $userAgent
+        $response = Invoke-WebRequest -Uri $linkUri -Method HEAD -UserAgent $userAgent -TimeoutSec 15
```

Comment:
Instead of hard-coding this value everywhere lets please add it as a new parameter and default the vault to 15 but folks can override it if needed.